Lib/test/test_abc.py

```diff
@@ -165,7 +165,7 @@ def method_one(self):
                 @abc.abstractmethod
                 def method_two(self):
                     pass
-            msg = r"class C without an implementation for abstract methods method_one, method_two"
+            msg = r"class C without an implementation for abstract methods 'method_one, method_two'"
```

Comment:
This is incorrect, we should quote each method individually.

Reply:
Thanks, done!